app/controllers/api/order_cycles_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class OrderCyclesController < BaseController
+    respond_to :json
+
+    def products
+      products = OpenFoodNetwork::ProductsRenderer.new(current_distributor, current_order_cycle, params).products_json
+
+      render json: products
+    rescue OpenFoodNetwork::ProductsRenderer::NoProducts
+      render status: :not_found, json: ''
+    end
+
+    def filters
```

Comment:
is there a better way we can do this in terms of API endpoints?
if we keep it we need at least a better name for this endpoint. "filters" is too specific to the shopfront, not directly related to the OrderCycle domain...
is there any reason to keep taxons and properties in the same endpoint? we can probably have two separate endpoints.

/api/order_cycles/:id/taxons - this is not really generic taxons, it's only product categories, so maybe:
/api/order_cycles/:id/categories

and then: /api/order_cycles/:id/properties

I think these are better names ☝️

Reply:
Yeah this bit is WIP. Its: "taxons and properties of variants distributed in this order cycle".

Reply:
Moved to separate endpoints :+1: